src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -1801,12 +1804,15 @@ public void addError(AuditEvent event) {
             final IndentComment comment = comments[position];
             position++;
 
+            final String possibleExceptedMessages = Arrays.stream(comment.getExpectedMessages())
+                    .reduce("", (cur, next) -> cur + "\"" + next + "\", ");
+            final boolean hasExpectedMessagePostfix =
+                    Arrays.stream(comment.getExpectedMessages()).anyMatch(message::endsWith);
             assertTrue(
                     "input expected warning #" + position + " at line " + comment.getLineNumber()
-                            + " to report '" + comment.getExpectedMessage() + "' but got instead: "
-                            + line + ": " + message,
-                    line == comment.getLineNumber()
-                            && message.endsWith(comment.getExpectedMessage()));
+                            + " to report one of the following: " + possibleExceptedMessages
+                            + "but got instead: " + line + ": " + message,
```

Comment:
I didn't realize this, but it still makes this assertion look off.
This can be ignored for now.